### ISSUE TITLE
After adding the mysql connection, the option is empty

### ISSUE THREAD

**Issue by qgxcry** [ErikEJ/EFCorePowerTools#193]
select "EF Core Power Tools" - "Reverse Engineer"  choose Database connection - Add Mysql，After adding the mysql connection, the option is empty
![image](https://user-images.githubusercontent.com/13807716/54586707-d3a69d00-4a58-11e9-8dd7-788a12a74a81.png)
@ErikEJ

**Comment by jespersh** [comment]
Did you install this? https://dev.mysql.com/downloads/windows/visualstudio/ 
The adding database connection interface for MySQL looks like this:
![image](https://user-images.githubusercontent.com/1004198/54588204-96271d00-4a22-11e9-84d8-c708e131527c.png)
The normal ODBC one won't do.

**Comment by qgxcry** [comment]
Yes，I have installed
@jespersh 
![image](https://user-images.githubusercontent.com/13807716/54590109-46684600-4a62-11e9-9805-9fe036e9eb23.png)

**Comment by ErikEJ** [comment]
What Power Tools version? You must use latest daily.

**Comment by qgxcry** [comment]
version 2.2.12 Reinstalling the latest version will not work either.

> What Power Tools version? You must use latest daily.

**Comment by jespersh** [comment]
> version 2.2.12 Reinstalling the latest version will not work either.
> 
> > What Power Tools version? You must use latest daily.

Mine is 2.2.26.0 which has the MySQL provider

**Comment by ErikEJ** [comment]
Thanks for stepping up, @jespersh  - yes, you need 2.2.26 or later!

**Comment by Bebu1985** [comment]
I have the same issue, but i am using the latest Version 2.2.35 and try to connect to an access database file. I also tried odbc, but the option field remains empty

**Comment by VaanRatsbane** [comment]
This situation is happening to me with the latest version, could the compatibility issue have resurfaced?
If so, are there download links available for previous versions? Thanks in advance.

**Comment by jespersh** [comment]
If you're using Visual Studio 2019, then you'll need this vsix: https://github.com/jespersh/mysql-for-visualstudio2019/releases

MySQL haven't updated their own yet...

**Comment by VaanRatsbane** [comment]
Yes I had that one installed (uninstalled and installed through that link just to be sure), it still happens

**Comment by jespersh** [comment]
I'll try it once I am home next week. Could you provide the efcorepowertools extension version? Should be something like 2.2.xx iirc

**Comment by VaanRatsbane** [comment]
2.2.63
Thank you

**Comment by leandrorovagnoli** [comment]
Same problem here.

Using the last version (EFCorePowerTools and Mysql for VS2019).

After adding MySql connection (which it says: "Test connection succeeded."), it still doesn't appear in the dropdown list.

**Comment by jespersh** [comment]
How are you adding the server? For some reason you need to add it through the Server Explorer:
![image](https://user-images.githubusercontent.com/1004198/63435382-19f89180-c427-11e9-8c80-adbe67656794.png)

Using the "Add" button in the reverse engineer button **doesn't work** for MySQL. Reason unknown:
![image](https://user-images.githubusercontent.com/1004198/63435513-51ffd480-c427-11e9-9977-3041e28cff04.png)

**Comment by jespersh** [comment]
I was able to identify the reason the Add button didn't work for MySQL. A pull request has been made to correct this issue.

Thanks for redirecting my attention to that

**Comment by VaanRatsbane** [comment]
Using the daily build I was able to properly add the connection and scaffold the database and context, thank you!

However, I noticed that the efpt renaming is not fully working. Single object properties are translated correctly, but the class names and collection properties still keep the original table names. I attempted this with a structurally identical database in another platform (Mssql) and it translated the names correctly, so I assume the issue is specific to Mysql.
